src/components/static/pages/Contact.tsx

```diff
@@ -28,6 +21,32 @@ const ContactInfo = styled.div`
   font-size: 18px;
 `;
 
+const ContactHero = styled.div`
```

Comment:
@roymoran  not sure if this should be called ContactHero? same with HeroContent, we got this from the Blog page so it could be unnecessary to name it this way for Contact page

Reply:
@cprivera this should work :)